# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieService.h
# Commit: 7a28b63b4fc3
# Full Hash: 7a28b63b4fc3c64e95e4e1885909731e2c2df524
# Author: Paul Zuehlcke <pbz@mozilla.com>
# Date: 2023-09-19 03:53:26
# Regressor Bug: 1839918
# File Overlap Count: 3
# Description:
#   Bug 1839918 - Pass BrowsingContext ID and third-party flag through CookieService. r=bvandersloot,cookie-reviewers,edgul
#   
#   Depends on D185302
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D185303
# ==============================================================================

diff -r 2bd8c91c5d97 -r 7a28b63b4fc3 netwerk/cookie/CookieService.h
--- a/netwerk/cookie/CookieService.h	Mon Sep 18 11:55:09 2023 +0000
+++ b/netwerk/cookie/CookieService.h	Mon Sep 18 11:55:09 2023 +0000
@@ -99,8 +99,8 @@
 
   bool SetCookiesFromIPC(const nsACString& aBaseDomain,
                          const OriginAttributes& aAttrs, nsIURI* aHostURI,
-                         bool aFromHttp,
-                         const nsTArray<CookieStruct>& aCookies);
+                         bool aFromHttp, const nsTArray<CookieStruct>& aCookies,
+                         uint64_t aBrowsingContextId, bool aIsThirdPartyCookie);
 
  protected:
   virtual ~CookieService();